### PR TITLE
docs: fix typo in `save-peer` description

### PR DESCRIPTION
**Pull request by chalkygames123**
Corrected the typo and then ran `TAP_SNAPSHOT=1 npm test`.

### CODE REVIEW

File: tap-snapshots/test/lib/commands/view.js.test.cjs [hunk 1]
```diff
@@ -82,7 +82,7 @@ dist
 dist-tags:
 [1m[32mlatest[39m[22m: 1.0.0
 
-published [33myesterday[39m
```
Comment (wraithgar) [inline tap-snapshots/test/lib/commands/view.js.test.cjs]: not sure why this changed ended up in your tests but it's making CI fail. Likely just need to revert this one to match`release-next`, and it's something for us to look into.

**Comment by wraithgar** [comment]
Try running `npm run snap` instead, there's a `postsnap` hook that runs that rebuilds additional files that are dependent on these config definitions.

**Comment by chalkygames123** [comment]
Okay, I'll do. Thanks for your instruction :)

**Comment by chalkygames123** [comment]
Hmm, while the changes now include `config.md`, CI still fails. Should I only commit the change in `definitions.js` for now?

**Comment by wraithgar** [comment]
I think if you omit the changes in ` tap-snapshots/test/lib/commands/view.js.test.cjs` you should be ok. If not we'll just call it ok and I'll fix it when I merge it and make sure it's passing.

**Comment by chalkygames123** [comment]
Now it looks ok!

**Review by wraithgar — APPROVED** [review]

**Comment by wraithgar** [comment]
Thanks for sticking with it, I know these tests are tricky to get passing but they save us a lot of headache down the line when it comes to cutting our releases.

**Pull request merged**